### DISCUSSION TITLE
Welcome to jira-code-review-ready Discussions!

### DISCUSSION THREAD

**Discussion by tonning** [HelixSleep/github-action-jira-status-check#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dabichosilva** [comment]
Hello @tonning ! Thanks for sharing this tool.
I tried implementing it but I'm always getting error with the Jira endpoint.
I configured a secret `JIRA_BASE_URL` trying to fix it, but no luck yet.
Am I missing something?

**Reply by dabichosilva** [reply]
Ok, after some research I found how to configure everything and made it work. But, it is not considering the argument I passed! I need to check the status is "Testing", but it left the default one ```Error: Status must be "Under Code Review". Found "Backlog".```
